package/MDAnalysis/analysis/base.py

```diff
@@ -219,6 +251,11 @@ def __getstate__(self):
     def __setstate__(self, state):
         self.data = state
 
+from typing import Callable, Iterable
+
+
+# from dask import delayed
```

Comment:
Imports should all go at the top of the file.

Reply:
done